subo/command/build.go

```diff
@@ -37,6 +38,15 @@ func BuildCmd() *cobra.Command {
 				return errors.New("🚫 no runnables found in current directory (no .runnable yaml files found)")
 			}
 
+			innerFiles, err := ioutil.ReadDir(bctx.Cwd)
+			if err != nil {
+				return errors.Wrapf(err, "Failed to list files in %s", bctx.Cwd)
+			}
+			fileName, _ := context.ContainsRunnableYaml(innerFiles)
+			if fileName == ".runnable.yaml" {
+				return errors.New("🚫 Cannot build inside a Runnable directory. Please run from project root.")
```

Comment:
Instead of returning an error, let's just print a warning that says "building single Runnable, will not create a bundle" or something like that. Technically building just one Runnable is valid.